src/data/countable/basic.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import logic.equiv.nat
+import logic.equiv.fin
+import data.countable.defs
+
+/-!
+# Countable `Sort*`s and `Type*`s
+
+In this file we define a typeclass saying that a given `Sort*` is countable. See also `encodable`
+for a version that singles out a specific encoding of elements of `α` by natural numbers.
+
```

Comment:
```suggestion
```
Matter of style, I guess.

Reply:
I prefer to have a blank line after a section title. It visually separates the title from the body.